pkg/cmd/experimental/router/router.go

```diff
@@ -93,22 +93,22 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out io.Writer)
 			case 1:
 				name = args[0]
 			default:
-				glog.Fatalf("You may pass zero or one arguments to provide a name for the router")
+				fmt.Errorf("You may pass zero or one arguments to provide a name for the router")
```

Comment:
The roughly equivalent command is `cmdutil.CheckErr`, but most static validation problems and some dynamic errors should provide a `cmdutil.UsageError`